x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.test.tsx

```diff
@@ -41,7 +41,7 @@ describe('RuleActionsOverflow', () => {
       const wrapper = shallow(
         <RuleActionsOverflow
           rule={mockRule('id')}
-          userHasNoPermissions={false}
+          userHasPermissions
```

Comment:
Within this test file when I run this:

```ts
node --trace-warnings scripts/jest.js x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.test.tsx
```

I get a complete test pass but I see these errors in the console:

```ts
(node:72170) UnhandledPromiseRejectionWarning: Error: Kibana services not initialized - are you trying to import this module from outside of the SIEM app?
    at emitUnhandledRejectionWarning (internal/process/promises.js:170:15)
    at processPromiseRejections (internal/process/promises.js:247:11)
    at processTicksAndRejections (internal/process/task_queues.js:96:32)
(node:72170) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:72170) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitDeprecationWarning (internal/process/promises.js:180:11)
    at processPromiseRejections (internal/process/promises.js:249:13)
    at processTicksAndRejections (internal/process/task_queues.js:96:32)
```

This might be a pre-existing condition, and I have been seeing more of these `UnhandledPromiseRejectionWarning` popping up even though the tests pass which isn't a good thing. It means we are doing something we shouldn't be in either the test or the code its self.

That error is coming from here:

https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/lib/kibana/services.ts#L44-L46

Unfortunately that error doesn't have a `console.trace()`, but once I add one to that file and location I see this additional stack trace:

```ts
onsole.error
    Trace: 
        at Function.throwUninitializedError (/Users/frankhassanabad/projects/kibana/x-pack/plugins/security_solution/public/common/lib/kibana/services.ts:44:13)
        at Function.get (/Users/frankhassanabad/projects/kibana/x-pack/plugins/security_solution/public/common/lib/kibana/services.ts:29:12)
        at exportRules (/Users/frankhassanabad/projects/kibana/x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/api.ts:327:25)
        at onClick (/Users/frankhassanabad/projects/kibana/x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx:103:36)
        at HTMLUnknownElement.callCallback (/Users/frankhassanabad/projects/kibana/node_modules/react-dom/cjs/react-dom.development.js:336:14)
        at HTMLUnknownElement.callTheUserObjectsOperation (/Users/frankhassanabad/projects/kibana/node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
        at innerInvokeEventListeners (/Users/frankhassanabad/projects/kibana/node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:318:25)
        at invokeEventListeners (/Users/frankhassanabad/projects/kibana/node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
        at HTMLUnknownElementImpl._dispatch (/Users/frankhassanabad/projects/kibana/node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
        at HTMLUnknownElementImpl.dispatchEvent (/Users/frankhassanabad/projects/kibana/node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
```

Which tells me we are trying to call into `KibanaServices.get.http` without a mock. Adding the mock like this to this file should remove the errors:

```ts
jest.mock('../../../../common/lib/kibana', () => {
  return {
    KibanaServices: {
      get: () => ({
        http: { fetch: jest.fn() },
      }),
    },
  };
});
```

Reply:
Yup, with the mock, the error is gone.